tests/docker/dns.bats

```diff
@@ -120,17 +120,17 @@ function teardown() {
      [[ "$output" == "$gateway" ]]
 
      # DNAT
-     docker exec "$syscont" sh -c "nft list chain nat DOCKER_OUTPUT | egrep -q \"tcp ip daddr ${gateway}\""
```

Comment:
I thought we were going to drop the "tcp" portion, rather than move it from prefix to suffix. Would moving it work on say Ubuntu and other distros?

Reply:
At first I tried to remove the tcp portion as we had discussed but then the next test also failed on line [126](https://github.com/nestybox/sysbox/pull/807/files/0359eeac8a8d5dfc55f429288ff7ec8bc440b443#diff-d169e0ea66bf3ac00550b0bb39baf710adb4cdb6b4cc92bbfd36b97a5f7964c5L126). 

While checking the output of the command that is being run I figured that the output looks like where by looking the output it looks like the words `tcp` and `udp` have been moved.
```bash
/ #
/ # nft list chain nat DOCKER_OUTPUT
table ip nat {
        chain DOCKER_OUTPUT {
                ip daddr 172.19.0.1 tcp dport 53 counter packets 0 bytes 0 # xt_DNAT
                ip daddr 172.19.0.1 udp dport 53 counter packets 0 bytes 0 # xt_DNAT
        }
}
```

In the tests `tcp` and `udp` are being tested separately and by removing the `tcp` word from the `egrep` command I wouldn't be able to ensure the proper testing of these.

Finally, regarding the final point about the different distros by checking the code it seems that these check are happening in an alpine docker container as shown below. So, unless I misunderstood something I would expect that to be compatible regardless the host distro (Ubuntu or others)
```
local syscont=$(docker_run --rm --net=usernet ${CTR_IMG_REPO}/alpine-docker-dbg:latest tail -f /dev/null)
```

Reply:
Makes sense, thanks!